locales/bn/bn.json

```diff
@@ -579,10 +579,10 @@
     "The :attribute must be at least :length characters and contain at least one uppercase character, one number, and one special character.": ":attribute অন্তত হতে হবে :length অক্ষর এবং অন্তত একটি য়ের বড়হাতের অক্ষর ছোটহাতের অক্ষর ছোটহাতের অক্ষর, একটি সংখ্যা, এবং একটি বিশেষ অক্ষর থাকতে হবে.",
     "The :attribute must be at least :length characters and contain at least one uppercase character.": ":attribute অন্তত :length অক্ষর হতে হবে এবং অন্তত একটি বড় হাতের অক্ষর ধারণ করতে হবে.",
     "The :attribute must be at least :length characters.": ":attribute অন্তত হতে হবে :length অক্ষর.",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`), but here - a period (`.`).

This translation was by machine.